lib/irc/ConnectionInstance.js

```diff
@@ -309,7 +309,7 @@ ConnectionInstance.create = function(server, opts, onCreatedCallback) {
         port: server.getPort(),
         selfSigned: server.useSslSelfSigned(),
         retryCount: 0,
-        family: server.getIpv6Prefix() ? 6 : 4,
+        family: server.getIpv6Prefix() || server.getIpv6Only() ? 6 : null,
```

Comment:
No, this breaks the whole purpose of this patch. This *has* to be `null` to not force to use IPv4.

Intended behaviour is:
* if there is `prefix` option present in `ipv6` section or `only` option set to `true`, force IPv6 connection;
* otherwise, use IPv6 if available, fall back to IPv4 if not, but *do not* force IPv4.